src/VAC/VectorAnimationComplex/VAC.h

```diff
@@ -158,7 +158,7 @@ class Q_VPAINT_EXPORT VAC: public SceneObject
     // -- Drag and drop --
     void prepareDragAndDrop(double x0, double y0, Time time);
     void performDragAndDrop(double x, double y);
-    void completeDragAndDrop();
+    void completeDragAndDrop(bool isEmitCheckpoint = true);
```

Comment:
"is" prefix is used only on getter functions, since we ask "is enabled?". There it is a command, so just name it "emitCheckpoint"

Reply:
Done